ci-operator/step-registry/aws/provision/network-firewall/rule-group/aws-provision-network-firewall-rule-group-commands.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION=${REGION:-$LEASED_RESOURCE}
+CAPACITY=${CAPACITY:-100}
+
+# get user arn
+# arn:aws:iam::301721915996:user/qe-jenkins
+STACK_NAME=$(cat "${SHARED_DIR}/firewall_stack_name")
+rule_group_name="${STACK_NAME}-rule-group"
+RULE_GROUP_ALLOWED=$(echo -e '
+{
+  "RulesSource": {
+    "RulesSourceList": {
+      "Targets": [
+        ".quay.io",
+        ".redhat.io",        
+        "api.openshift.com",
+        "access.redhat.com",
+        "sso.redhat.com",        
+        ".amazonaws.com"
```

Comment:
Just set some domains and need to check with the feature owners about what should be here.